src/react/src/util/styles.js

```diff
@@ -1652,11 +1652,11 @@ export const claimedFacilitiesDetailsStyles = () =>
 
 export const claimFacilitiesListStyle = Object.freeze({
     searchButton: Object.freeze({
-        margin: '10px 0',
+        margin: '16px 0',
     }),
     bodyStyle: Object.freeze({
         padding: '10px 0',
-        fontSize: 'calc(2* var(--rem))',
+        fontSize: '16pt',
```

Comment:
We don't use `pt` units in the project. Why it is not `px`?

Reply:
+1